spec/controllers/skills_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe SkillsController do
+  describe "managing skills" do
+  
+    before do
+      Skill.create(skill: "Rspec")
+      expect(Skill.find_by(skill: 'Rspec')).not_to be_nil
+    end
+
+    context "adding skills" do
+      it "can add an skill" do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context "editing skills" do
+      it "can edit an skill" do
+        skills = Skill.find_by(skill: 'Rspec')
+        skills.skill = 'Business Development'
+        skills.save
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context "destroy skills"  do
+      it "can destroy an skill" do
+        @skill = Skill.find_by(skill: 'Rspec').destroy
+        expect(Skill.find_by(skill: 'Rspec')).to be_nil
```

Comment:
looks like you did both here :) You can just `expect(@skill).to be_nil`... You're not using @skill now :)

Reply:
So when I do it like that. The Skill isn't really destroyed so the test fails. 

```context "destroy skills"  do
      it "can destroy an skill" do
        @skill = Skill.find_by(skill: 'Rspec').destroy
        expect(@skill).to be_nil
      end
    end```

Reply:
So I expect that when I find the skill 'Rspec', it will be nil because it's deleted. I guess I can take @skill out now.

Reply:
Oh right, my bad! The instance will still hold the values!
I think you'll want to:
```ruby
Skill.find_by(skill: 'Rspec').destroy
@skill = Skill.find_by(skill: 'Rspec')
expect(@skill).to be_nil
```